staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -560,6 +560,20 @@ type DeleteOptions struct {
 	// +optional
 	// +listType=atomic
 	DryRun []string `json:"dryRun,omitempty" protobuf:"bytes,5,rep,name=dryRun"`
+
+	// IgnoreStoreReadErrorWithClusterBreakingPotential, if set to true,
```

Comment:
nit: the style here in the struct is to not mention the field name as first word.